src/main/java/bb/example/services/Validators.java

```diff
@@ -0,0 +1,15 @@
+package bb.example.services;
+
+public class Validators {
+    public static boolean isValidNumber(String input) {
+        return !input.isBlank() && input.chars().allMatch(Character::isDigit);
+    }
+
+    public static boolean doesContainChar(String input) {
+        return (input.chars().allMatch(letter -> Character.isAlphabetic(letter) || letter == ' ' || letter == '-') && !input.isBlank());
+    }
+
+    public static boolean doesContainRangeOfDigits(int age) {
```

Comment:
Perhaps "isAgeValid()" wouldeb a better name? Also, there are people older than 100 years and the age of 0 should also be accepted.